tutorials/text/text_intro.py

```diff
@@ -38,17 +38,20 @@
 `.pyplot` API       OO API              description
 =================== =================== ======================================
 `~.pyplot.text`     `~.Axes.text`       Add text at an arbitrary location of
-                                        the `.Axes`.
+                                        the `~matplotlib.axes.Axes`.
```

Comment:
You should be able to do `~.axes.Axes` that saves a bit of space (and similarly for the changes below)

Reply:
Probably. But due to three different `Axes` being around I wanted to be on the safe side first. ;-)

Reply:
Umm, OK, so does this change *break* the simple `.Axes` shortcut?  If so, I'm a little leery about accepting it, because I can't imagine there aren't other places in the code where this shortcut has been used then the ones you've flagged above....